internal/errs/errs.go

```diff
@@ -258,3 +278,14 @@ func Unpack(err error) []error {
 	}
 	return result
 }
+
+// IsUserFacing identifies whether this error was curated for end-users and
+// returns that error. This is NOT the same as an error that is localized.
+func IsUserFacing(err error) (error, bool) {
+	var userFacingError *UserFacingError
```

Comment:
Please use an interface instead. We don't want to require the UserFacingError type itself, the type is just for convenience. It should be possible for us to construct our own user facing error type.